test/api/contenttype-query.spec.ts

```diff
@@ -1,11 +1,14 @@
+// @ts-nocheck
 import { ContentTypeQuery } from '../../src/lib/contenttype-query';
 import { stackInstance } from '../utils/stack-instance';
-import { TContentTypes } from './types';
+import { TContentType, TContentTypes } from './types';
 
 const stack = stackInstance();
 describe('ContentTypeQuery API test cases', () => {
-  it('should check for content_types are defined', async () => {
-    const result = await makeContentTypeQuery().find<TContentTypes>();
+  it.only('should check for content_types are defined', async () => {
+    const result = await makeContentTypeQuery().find<TContentType>();
+    console.log('🚀 ~ it.only ~ result:', result.content_types?.length);
```

Comment:
please remove this console if it is not required

Reply:
Removed this